### PR TITLE
Update to handle .csv USGS inventory files in lieu of .json

### PR DESCRIPTION
**Pull request by HicksMatthew**
The code was updated to handle the change from inventory files being .json to .csv as well as updated to handle changes to some of the column formats.

### CODE REVIEW

File: lambdas/subset_granules.py [hunk 1]
```diff
@@ -1,26 +1,132 @@
...
+
+
+def build_landsat_s3_path(granule):
+    scene_id = granule["landsat_product_id"]
+    if len(scene_id) < 40:
+        pass
+    meta = landsat_parse_scene_id(scene_id)
+    if meta["sensor"] != "C":
+        raise NameError("USGS Landsat Scene sensor is not OLI-TIRS.\n")
+    ls_s3_path = (
```
Comment (sharkinsspatial) [inline lambdas/subset_granules.py:91]: Use f strings over string concat.

File: lambdas/subset_granules.py [hunk 2]
```diff
@@ -1,26 +1,132 @@
...
-        " s.processing_level = '1' AND"
+        Expression="select * from S3Object s where"
+        " TO_TIMESTAMP(s.date_acquired,'yyyy-MM-dd HH:mm:ss') BETWEEN"
+        f" TO_TIMESTAMP('{start_date}','yyyy-MM-dd HH:mm:ss') AND"
+        f" TO_TIMESTAMP('{end_date}','yyyy-MM-dd HH:mm:ss') AND"
+        " s.processing_level_short = '1' AND"
         " s.sensor_id = 'OLI_TIRS' AND"
-        " s.spacecraft_id = 'LANDSAT_8' AND"
-        " s.product_id LIKE '%_T1'",
+        " s.landsat_product_id LIKE '%_T1' AND"
```
Comment (sharkinsspatial) [inline lambdas/subset_granules.py:117]: Include optional Landsat 8/9 filter to support Landsat 9 reprocessing use case.

**Review by sharkinsspatial — CHANGES_REQUESTED** [review]

File: lambdas/subset_granules.py (hunk 1 above)
Comment (HicksMatthew) [reply lambdas/subset_granules.py:91]: Implemented and tested as f string

File: lambdas/subset_granules.py (hunk 2 above)
Comment (HicksMatthew) [reply lambdas/subset_granules.py:117]: Added environment variable to set Landsat 8 or 9 for historical processing.

**Review by sharkinsspatial — APPROVED** [review]

**Pull request merged**